### PR TITLE
Add registered trademark to home page title

### PR DESCRIPTION
**Pull request by umarmughal824**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
  - [ ] Tag @ferdi or @pdpinch for review
- [ ] Migrations
  - [ ] Migration is backwards-compatible with current production code
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested
- [ ] Settings
  - [ ] New settings are documented and present in `app.json`
  - [ ] New settings have reasonable development defaults, if applicable

#### What are the relevant tickets?
#4476 / https://trello.com/c/QDQ0zPEH/43-home-page-title

#### What's this PR do?
Add registered trademark to the home page title, header

#### How should this be manually tested?
Just visit the home page.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
Screenshot 2020-03-30 at 19 35 14

#### What GIF best describes this PR or how it makes you feel?
(Optional)

### CODE REVIEW

**Comment by pdpinch** [comment]
Sorry this wasn't clear, but the issue was just to change the HTML _title_ 

The other text has been reviewed by legal once already, and we don't want to change it again.

**Comment by umarmughal824** [comment]
> Sorry this wasn't clear, but the issue was just to change the HTML _title_
> 
> The other text has been reviewed by legal once already, and we don't want to change it again.

ok will revert for the remaining.

**Comment by umarmughal824** [comment]
@pdpinch I'm waiting for your comments here as you mentioned in the standup meeting that you want to review it once again.

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -145,7 +145,7 @@ <h4>Pursue a Master’s</h4>
         </div>
       </section>
       <section class="row current-programs-section">
-        <h3>MITx MicroMasters Programs</h3>
+        <h3>MITx MicroMasters&reg; Programs</h3>
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]: The mark is only required once on the page, and the first appearance is above this one. Please remove this.

File: cms/templates/cms/home_page.html [hunk 2]
```diff
@@ -68,7 +68,7 @@
         {% endif %}
         <div class="banner-wrapper-content">
           <h1>
-            MITx MicroMasters Programs: Bringing MIT to you
+            MITx MicroMasters&reg; Programs: Bringing MIT to you
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]: We only need this mark if it helps change the organic search results. Let's remove it for now, and I'll keep an eye on how Google responds. If we need it, we can add it later.

**Pull request merged**